packages/web/stores/index.tsx

```diff
@@ -22,7 +21,12 @@ export const StoreProvider: FunctionComponent = ({ children }) => {
       new RootStore({
         txEvents: {
           onBroadcastFailed: () => invalidateQueryData(apiUtils),
-          onFulfill: () => invalidateQueryData(apiUtils),
+          onFulfill: () => {
+            invalidateQueryData(apiUtils);
+            // NOTIFI QUESTION - We need a way to let the Notifi card know that the transaction has been fulfilled.
```

Comment:
@jonator we couldn't relay from here, as it'd require a Notifi context to wrap the StoreProvider